Sources/APIProvider.swift

```diff
@@ -8,6 +8,8 @@
 import Foundation
 import Alamofire
 
+public typealias Handler<T> = (Result<T>) -> Void
```

Comment:
I would rename this to `RequestCompletionHandler` to be more explicit

Reply:
Indeed